statics/js/components/rule-detail.js

```diff
@@ -666,45 +718,115 @@ Vue.component('rule-detail', {
 
   beforeDestroy: function () {
     this.unwatch();
-    this.graph.removeHandler(this.handler);
   },
 
   mounted: function () {
-    var self = this;
-    var handle = function(e) {
-      if (! self.bridge) return;
-      var tgtType = e.target.metadata.Type;
-      if ((tgtType === 'ofrule' || tgtType === 'ofgroup') && e.source.id == self.bridge.id ) {
-        self.memoBridgeLayout = null;
+        var self = this;
+        var handle = function(e) {
+            if (! self.bridge) return;
+            if (e.target.metadata.Type === 'ofrule' && e.source.id == self.bridge.id ) {
+                self.getRules();
+            }
+        };
+        var handleUpdate = function(n) {
+            if (n.metadata.Type == 'ofrule') {
+                self.getRules();
+                var tgtType = e.target.metadata.Type;
+                if ((tgtType === 'ofrule' || tgtType === 'ofgroup') && e.source.id == self.bridge.id ) {
+                    self.memoBridgeLayout = null;
+                }
+            }    
+        };
+        var handleUpdate = function(n) {
+            if ((n.metadata.Type == 'ofgroup' && self.memoBridgeLayout.groupsUUID.has(n.metadata.UUID)) ||
+                (n.metadata.Type == 'ofrule' && self.memoBridgeLayout.rulesUUID.has(n.metadata.UUID))) {
+                self.memoBridgeLayout = null;
+            }
+        };
+        this.handler = {
+        onEdgeAdded: handle,
+        onEdgeDeleted: handle,
+        onNodeUpdated:handleUpdate
+        };
+        this.realgraph.addHandler(this.handler);
```

Comment:
You changed according to my comment thx, but I'm not sure this is still working ? I think more modification are required